### PR TITLE
GPX is now supported

### PR DESCRIPTION
**Pull request by rtsisyk**

### CODE REVIEW

File: content/_index.ca.md [hunk 1]
```diff
@@ -36,8 +36,7 @@ excursionistes i ciclistes:
 - Línies de contorn, perfils d'elevació, pics i pendents
 - Navegació pas a pas a peu, amb bicicleta i amb cotxe amb guia de veu
 - Cerques ràpides al mapa, sense connexió
-- Els marcadors s'exporten i importen en els formats KML/KMZ (GPX està
-  [planificat][gpx_issue])
+- Els marcadors s'exporten i importen en els formats KML/KMZ/GPX
```
Comment (biodranik) [inline content/_index.ca.md:39]: @rtsisyk  We only support GPX import now, not export. Do you have any ideas on how to change these texts?

**Review by biodranik — CHANGES_REQUESTED** [review]
@rtsisyk Thanks! One important comment.

File: content/_index.ca.md (hunk 1 above)
Comment (rtsisyk) [reply content/_index.ca.md:39]: This is sad... What is about:

> Bookmarks export and import in KML/KMZ/GPX formats

=>

> Saved places and tracks in KML/KMZ/GPX formats

**Comment by rtsisyk** [comment]
Updated to "Bookmarks and tracks in KML, KMZ, GPX formats".

**Pull request closed without merging**